BlueBrain/HighFive
# Soft | External links (minimal API)

**Description**

- [x] Added minimal API to create Soft and External links

Fixes #154

**How to test this?**

See tests HighFiveSoftLink, HighFiveardHLink, HighFiveExternalLink

```bash
cmake ..
make -j8
make test
```

**Test System**
 - OS: [e.g. Ubuntu 20.04]
 - Compiler: [e.g. clang 12.0.0]
 - Dependency versions: [e.g. hdf5 1.12]
